libglusterfs/src/glusterfs/glfs-message-id.h

```diff
@@ -190,11 +222,10 @@
 #define GLFS_NAMES(_num, _fields...) GLFS_APPLY(GLFS_NAME, (), _num, ##_fields)
 
 /* Generate a format string for all fields. */
-#define GLFS_FMTS(_num, _fields...)                                            \
-    GLFS_APPLY(GLFS_FMT, (", "), _num, ##_fields)
+#define GLFS_FMTS(_num, _fields...) GLFS_APPLY(GLFS_FMT, (", "), _num, ##_fields)
 
 /* Generate the function call argument declaration for all fields. */
-#define GLFS_ARGS(_num, _fields...) GLFS_APPLY(GLFS_ARG, (, ), _num, ##_fields)
+#define GLFS_ARGS(_num, _fields...) GLFS_APPLY_ARGS(GLFS_ARG, (, ), _num, ##_fields)
```

Comment:
Why is this new macro required ?

Reply:
well....  if you simply add 'void' to the existing GLFS_APPLY_0 macro to get a valid C99 fn decl in the places where it's needed, then you also get 'void' added in other places where it's an error.  compare the GLFS_APPLY_0 and GLFS_APPLY_ARGS_0 macros.

Then look at how the GLFS_APPLY_*, GLFS_APPLY, and GLFS_ARG macros work and where they get expanded.

Reply:
Understood. Thanks !